src/Network/Google/Drive/File.hs

```diff
@@ -64,6 +67,7 @@ data FileData = FileData
     , fileSize :: !(Maybe Int)
     , fileDownloadUrl :: !(Maybe Text)
     , fileMimeType :: !Text
```

Comment:
I would re-use the type synonym you've added here (and anywhere else `fileMimeType` is used).